muikku-core-plugins/src/main/resources/META-INF/resources/scripts/src/@types/pedagogy-form.ts

```diff
@@ -6,6 +6,7 @@ const useRoles = [
   "COURSE_TEACHER",
   "GUIDANCE_COUNSELOR",
   "SPECIAL_ED_TEACHER",
+  "STUDENT_PARENT",
```

Comment:
Yes, I would suggest that too, but then there's a divide between backend and frontend in naming this this role. Naming it like this is false, because it's not always the parent who is in this role. Change it everywhere to be consistent or leave it like this, is how I see it.